config/initializers/filter_parameter_logging.rb

```diff
@@ -1,4 +1,4 @@
 # Be sure to restart your server when you modify this file.
 
-# Configure sensitive parameters which will be filtered from the log file.
-Rails.application.config.filter_parameters += %I[password email first_name last_name]
+# Configure parameters to be filtered from the log file. Use this to limit dissemination of
```

Comment:
of?

Reply:
not sure what happened, reverted to original comment